GitExtUtils/ServiceContainerRegistry.cs

```diff
@@ -0,0 +1,13 @@
+using System.ComponentModel.Design;
+using CommunityToolkit.Mvvm.Messaging;
+
+namespace GitExtUtils;
+
+public static class ServiceContainerRegistry
+{
+    public static void RegisterServices(ServiceContainer serviceContainer)
+    {
+        IMessenger messenger = WeakReferenceMessenger.Default;
+        serviceContainer.AddService(messenger);
```

Comment:
The `messenger` instance will be used for additional registrations in the near future.